.github/actions/docker-build-backend/action.yml

```diff
@@ -18,6 +18,10 @@ runs:
     - name: Set up docker buildx
       uses: docker/setup-buildx-action@v3
 
+    - name: Prepare docker build inputs
+      shell: bash
+      run: scripts/docker-build.pre
```

Comment:
The docker build gets commit and tag data as input now.  The command above puts this information into files given to docker.